komodor/komodormanifest.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Can we please rename this file to `/manifest.json`? It should be in the root of the `komodor` check directory